sql/item_mods.sql

```diff
@@ -76194,6 +76283,7 @@ INSERT INTO `item_mods` VALUES (28266,29,5);    -- MDEF: 5
 INSERT INTO `item_mods` VALUES (28266,31,69);   -- MEVA: 69
 INSERT INTO `item_mods` VALUES (28266,68,92);   -- EVA: 92
 INSERT INTO `item_mods` VALUES (28266,384,400); -- HASTE_GEAR: 400
+INSERT INTO `item_mods` VALUES (28266,1011,2);  -- PFLUG: 2
```

Comment:
Just a heads up, looks like this exists, but is not wired up

Reply:
Not surprising, and sadly, not something I can really contribute (The wired up part).